### PR TITLE
Preview app

### PR DESCRIPTION
**Pull request by ayinloya**
This update uses [sst](https://sst.dev) to setup preview app that can be used for testing web-client changes.
This is similar to the example app we currently use.
The only difference being that, we are now able to share a link to the preview app for testing on mobile.

# Test Instructions
The readme has instructions to setup and run the app locally.
Confirm that you are able to create a preview app locally.

For the preview link in PRs.
Confirm a review link is created once the pr is opened.
and deleted once the pr is closed
Confirm that the link is accessible and jobs can be run successfully.

NOTE: to set up locally, you'll need aws credentials with the right access/roles

### CODE REVIEW

File: previews/app/routes/products.$productName.tsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    if (typeof window.SmileIdentity === 'function' && config) {
+      window.SmileIdentity({
+        ...config,
+        document_ids: [config.document_id],
+        document_capture_modes: (
+          (config.document_capture_modes as String) ?? ''
+        ).split(','),
+        partner_details: {
+          partner_id: config.partner_id,
+          name: 'Demo Account',
```
Comment (beaglebets) [inline previews/app/routes/products.$productName.tsx:90]: why not allow actual values to be passed in via the env variables?

File: previews/default_app/routes/_index.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  return [
+    { title: 'New Remix App' },
+    { name: 'description', content: 'Welcome to Remix!' },
+  ];
+};
+
+export default function Index() {
+  return (
+    <div style={{ fontFamily: 'system-ui, sans-serif', lineHeight: '1.8' }}>
+      <h1>Welcome to Remix</h1>
```
Comment (beaglebets) [inline previews/default_app/routes/_index.tsx:13]: Not sure about this line.

**Review by beaglebets — COMMENTED** [review]
I think this is really close

File: previews/default_app/routes/_index.tsx (hunk 2 above)
Comment (ayinloya) [reply previews/default_app/routes/_index.tsx:13]: from the initial template used, will update

**Pull request merged**